avaframe/ana5Utils/distanceTimeAnalysis.py

```diff
@@ -580,7 +581,14 @@ def initializeRangeTime(modName, cfg, dem, simHash):
     """
 
     # fetch configuration and add info
-    cfgRangeTime = cfgUtils.getModuleConfig(modName)
+    if cfg.has_section('distanceTimeAnalysis_override'):
```

Comment:
mh, I guess this is up for discussion, but should we really make these special treatments? Or would it be possible to use the getModuleConfig instead?

Reply:
removed